runelite-client/src/main/java/net/runelite/client/ui/overlay/infobox/InfoBoxOverlay.java

```diff
@@ -72,8 +73,11 @@ public Dimension render(Graphics2D graphics, Point parent)
 		int width = infoBoxes.size() * (BOXSIZE + SEPARATOR);
 		int x = 0;
 
-		for (InfoBox box : infoBoxes)
```

Comment:
Why was this changed?

Reply:
Oops, that should not have been added. Fixed.